packages/native/src/useLinking.native.tsx

```diff
@@ -58,8 +59,20 @@ export default function useLinking(
 
   const extractPathFromURL = React.useCallback((url: string) => {
     for (const prefix of prefixesRef.current) {
-      if (url.startsWith(prefix)) {
-        return url.replace(prefix, '');
+      /**
+       * https is default schema, as mentioned in android documentation
+       * User can add prefixes like this `example.com`
+       */
+      const protocol = prefix.match(/^[^:]+:\/\//)?.[0] || 'https://';
```

Comment:
I would prefer no to add a schema by default unless user has specified it.
```suggestion
      /**
      const protocol = prefix.match(/^[^:]+:\/\//)?.[0] ?? '';
```